app/models/expense.rb

```diff
@@ -102,8 +102,7 @@ def expense_reason
   end
 
   def allow_reason_text?
-    return false if self.reason_id.nil?
-    expense_reason.allow_explanatory_text?
+    !!expense_reason&.allow_explanatory_text?
```

Comment:
This will fix sentry exception https://sentry.service.dsd.io/mojds/private-beta/group/1499/
I run into that error when uploading a JSON claim to test the scheme filter so I fix it here.